src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -47,17 +47,24 @@ public ProgramSubject(FailureStrategy failureStrategy, ProgramSubject.Program su
   public void generatesDeclarations(Boolean withExterns, String expected) {
     Options opts = new Options(!withExterns);
 
+
+
     List<SourceFile> sourceFiles = new ArrayList<>();
+    sourceFiles.add(SourceFile.fromFile("src/resources/base.js", UTF_8));
```

Comment:
added. Also had to move base into its own test, which is closer to reality usage.